src/content_working_group/lib.rs

```diff
@@ -1896,7 +1893,52 @@ decl_module! {
             // Ensure root is origin
             ensure_root(origin)?;
 
-            Self::do_set_lead(member, role_account)?;
+            // Ensure there is no current lead
```

Comment:
Why was this change made? I may be a good idea, but not clear what was the context for this.

Reply:
I originally had do_set_lead() because it needed to be called from genesis construction and from the extrinsic dispatch. When we decided against setting a lead at genesis I removed it.